com.unity.render-pipelines.high-definition/Editor/Material/MaterialHeaderScope.cs

```diff
@@ -66,6 +76,7 @@ public MaterialHeaderScope(string title, uint bitExpanded, MaterialEditor materi
                 ++EditorGUI.indentLevel;
         }
 
+        /// <summary>Dispose</summary>
```

Comment:
Disposes of the [**Insert here whatever this is**] and cleans up any resources it used.

Reply:
Side note for this: users don't see this as the function is private and this doc doesn't end up in the doc website but our doc validation tool requires us to document function implemented from interfaces, even if they are inaccessible